### PR TITLE
Slash Attachments

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

File: discord/types/interactions.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
 from __future__ import annotations
 
 from typing import Optional, TYPE_CHECKING, Dict, TypedDict, Union, List, Literal
+
+from discord.types.message import Attachment
```
Comment (nerdguyahmad) [inline discord/types/interactions.py:30]:
```suggestion
from .message import Attachment
```

**Comment by Lulalaby** [comment]
@nerdguyahmad could you implement the rest?

**Comment by nerdguyahmad** [comment]
> @nerdguyahmad could you implement the rest?

Sure.

**Comment by Lulalaby** [comment]
I'm too unfamiliar with it sadly.
correct implementation is here:
https://github.com/Aiko-IT-Systems/Discord-API-Docs/commit/4c28ec59a666a668456d3412e9ecde97e0f4220f
I can't pr it to dapi docs because of drama. There is a pr, but it's incorrect.

Attachments are missing the ephemeral flag

**Comment by Dorukyum** [comment]
Could a checklist for what's left on this be added?

**Comment by Dorukyum** [comment]
Update: added parsing for received attachments, tested with a txt file
Ready for review.

**Comment by Lulalaby** [comment]
Thanks ❤️

**Review by krittick — APPROVED** [review]

**Review by Middledot — APPROVED** [review]

**Pull request merged**